Index: Project Stage 3/sourceFiles/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import java.util.Calendar;\r\nimport java.util.Date;\r\n\r\npublic class Main {\r\n\r\n\r\n\r\n    public static void main(String[] args) {\r\n\r\n\r\n        menu menu = new menu();\r\n        menu.mainMenu();\r\n\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Project Stage 3/sourceFiles/Main.java b/Project Stage 3/sourceFiles/Main.java
--- a/Project Stage 3/sourceFiles/Main.java	(revision 73b422e57bb91878ec51e96fab0e4c3c98a2938e)
+++ b/Project Stage 3/sourceFiles/Main.java	(date 1668787639784)
@@ -1,6 +1,3 @@
-import java.util.Calendar;
-import java.util.Date;
-
 public class Main {
 
 
